frontend/webpack.config.ts

```diff
@@ -304,42 +304,57 @@ const config: Configuration = {
         /vendor-patternfly-4-shared~main.*\.css/,
       ],
     }),
-    new HtmlWebpackExcludeAssetsPlugin(),
```

Comment:
Removed as it no longer works with newer versions of the `HtmlWebpackPlugin`: see https://github.com/jamesjieye/html-webpack-exclude-assets-plugin/issues/18